java/memory/src/main/java/org/apache/arrow/memory/BaseAllocator.java

```diff
@@ -319,15 +318,6 @@ public ArrowBuf buffer(final long initialRequestSize, BufferManager manager) {
       listener.onAllocation(actualRequestSize);
       return buffer;
     } catch (OutOfMemoryError e) {
-      /*
```

Comment:
should this be moved to someplace netty specific?

Reply:
Thanks for the good suggestion. I have moved it into class PooledByteBufAllocatorL.